File: src/VacancyManager/VacancyManager/Scripts/VM/Admin/model/InputMessage.js
Ext.define('VM.model.InputMessage',
    {
        extend: 'Ext.data.Model',
        idProperty: 'Id',
        fields: [
            { name: 'Id', type: 'int' },
            { name: 'SendDate', type: 'date', dateFormat: 'MS' },
            { name: 'DeliveryDate', type: 'date', dateFormat: 'MS' },
            { name: 'Sender', type: 'string' },
            { name: 'Subject', type: 'string' },
            { name: 'Text', type: 'string' },
            { name: 'IsRead', type: 'boolean' },
            { name: 'Vacancy', type: 'string' },
            { name: 'ConsiderationId', type: 'int' },
            'AttachmentFile'
        ],

        proxy:
        {
            type: 'ajax',
            api:
            {
                read: '/InputMessage/Index',
                create: '/InputMessage/Create',
                update: '/InputMessage/Update',
                destroy: '/InputMessage/Delete'
            },

            reader:
            {
                type: 'json',
                root: 'data',
                successProperty: 'success'
            },

            writer:
            {
                type: 'json',
                encode: false,
                listful: true,
                writeAllFields: true,
                getRecordData: function (record) {
                    return { 'data': Ext.JSON.encode(record.data)
                    };
                }
            },
            headers: { 'Content-Type': 'application/json; charset=UTF-8' }
        }
    })

File: src/VacancyManager/VacancyManager/Models/InputMessage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace VacancyManager.Models
{
    public class InputMessage
    {
        [Key]
        public int Id { get; set; }

        public DateTime SendDate { get; set; }
        public DateTime DeliveryDate { get; set; }

        public string Sender { get; set; }
        public string Subject { get; set; }
        public string Text { get; set; }

        public bool IsRead { get; set; }

        public Nullable<int> ConsiderationId { get; set; }

        public virtual ICollection<Attachment> Attachments { get; set; }
    }
}